bl31/aarch64/crash_reporting.S

```diff
@@ -338,6 +339,7 @@ func do_crash_reporting
 	mrs	x8, cntkctl_el1
 	mrs	x9, fpexc32_el2
 	mrs	x10, sp_el0
+	mrs	x11, isr_el1
```

Comment:
minor: Strictly speaking this is a separate logical change and should be in a different commit. It at least needs a mention in the commit message